shoop/core/taxing/_tax_summary.py

```diff
@@ -16,7 +16,14 @@
 
 class TaxSummary(list):
     @classmethod
-    def from_line_taxes(cls, line_taxes, untaxed=TaxlessPrice(0)):
+    def from_line_taxes(cls, line_taxes, untaxed=None):
+        """
+        :type line_taxes: Iterable[LineTax]
+        :type untaxed: TaxlessPrice
+        :rtype: TaxSummary
+        """
+        if untaxed is None:
+            untaxed = TaxlessPrice(0)
```

Comment:
This should probably be fixed in the Price-object initialization, which uses the settings. It was a bad idea to use settings.SHOOP_HOME_CURRENCY in the constructor. The constructor should just set the self._currency to None if one is not given and the Price class should have a property getter for currency, which will return settings.SHOOP_HOME_CURRENCY when self._currency is None and self._currency otherwise. What do you think?